aws/sqs/terraform.tf

```diff
@@ -0,0 +1,3 @@
+terraform {
+  experiments = [module_variable_optional_attrs]
+}
```

Comment:
this file doesn't seem necessary (I don't see any use of the `optional` type

Reply:
I was using `defaults` earlier. I'll remove that.